index.html

```diff
@@ -26,6 +26,165 @@ <h1 class="career">I'm a software developer</h1>
                 <li><a class="twitter" href="https://twitter.com/Juanse77930"><img src="images/twitter.svg" alt="twitter"></a></li>
             </ul>
         </section>
+        <section class="whole-works-section">
+            <section class="works-title">
+                <a class="works"><img src="Backgroud-images/Disabled.png" alt="works-button"></a>
+                <h2 class="recent-works">My Recent Works</h2>
+                <img class="separation-bar" src="Backgroud-images/Indicator.png" alt="separation-bar">
+            </section>
+            <section class="works-container">
+                <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+                <div class="multi">
+                    <section class="multi-post">
+                        <h2 class="multi-post-title">Multi-Post Stories Gain+Glory</h2>
+                        <section class="buttons">
+                            <button class="languages">Ruby on rails</button>
+                            <button class="languages">css</button>
+                            <button class="languages">JavScript</button>
+                            <button class="languages">html</button>
+                        </section>
+                        <button class="see-project">See Project</button>
+                    </section>
+                </div>
+            </section>
+            <section class="works-container">
+                <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+                <div class="multi">
```

Comment:
- [ ] OPTIONAL : On some devices these images are broken, because the path does not have dot notation at the beginning. I suggest you to use dot slash notation to indicate path of files. Eg: 
`<img class="image-grey" src="./Backgroud-images/Img_Plaholder.png" alt="image">`